Source/LibationWinForms/Form1.ScanAuto.cs

```diff
@@ -17,11 +17,8 @@ public partial class Form1
 		private void Configure_ScanAuto()
         {
 			// creating InterruptableTimer inside 'Configure_' is a break from the pattern. As long as no one else needs to access or subscribe to it, this is ok
-			var hours = 0;
-			var minutes = 5;
-			var seconds = 0;
-			var _5_minutes = new TimeSpan(hours, minutes, seconds);
-			autoScanTimer = new InterruptableTimer(_5_minutes);
+
+			autoScanTimer = new InterruptableTimer(TimeSpan.FromMinutes(5));
```

Comment:
Wudja look at that. That's a handy little utility method I feel I really should have known about.